docker-compose.yml

```diff
@@ -20,6 +20,28 @@ services:
     tty: true
     depends_on:
       - db
+  migration:
```

Comment:
What are these new `migration` and `seed` entries about?  Should we just make those part of the Dockerfile and then rerun on an existing container via `docker run` if we want to update them in place?

Reply:
Oh man, this was me fighting hard with Docker. I attempted to do this in an existing container but this seemed problematic. 

Jake brought up a good question last night though - "Why Docker" I'm glad he said this because I wasn't sure how tough Elixir & Phoenix would be with Windows. He seemed to not have any problems so 🎉 ! That said, we might want to revisit this and either scrap the Docker setup, or only run Docker for CI.